examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -52,14 +52,15 @@ def act(self, obs: Observation) -> Tuple[float, float]:
         if not exp:
             return (0.0, 0.0)
         cur_state = obs.ego_vehicle_state
+
         assert math.isclose(
-            cur_state.heading, exp["heading"], abs_tol=1e-09
+            cur_state.heading, exp["heading"], abs_tol=1e-2
         ), f'vid={self._vehicle_id}: {cur_state.heading} != {exp["heading"]} @ {obs_time}'
         # Note: the other checks can't be as tight b/c we lose some accuracy (due to angular acceleration)
         # by converting the acceleration vector to a scalar in the observation script,
         # which compounds over time throughout the simulation.
         assert math.isclose(
-            cur_state.speed, exp["speed"], abs_tol=0.1
+            cur_state.speed, exp["speed"], abs_tol=1
```

Comment:
`abs_tol=1` means we can be a whole `m/s` off which is quite a bit since it compounds.

Reply:
I think it should be 0.1. We must have missed that.